templates/visualstudiocodeserver/index.ts

```diff
@@ -0,0 +1,33 @@
+import { Output, randomPassword, Services } from "~templates-utils";
+import { Input } from "./meta";
+
+export function generate(input: Input): Output {
+  const services: Services = [];
+  const databasePassword = randomPassword();
```

Comment:
this is unused

Reply:
remove it, can send a pr in about 8 hours after work

Reply:
Fixed